app/style.css

```diff
@@ -102,6 +124,7 @@
 }
 
 .btn, .success{
```

Comment:
```css
selector1,
selector2,
selectorN {
  // rule
}
```
You can add stylelint to enforce this and other rules.

Reply:
good idea, will have to wait until I have time to read it more.